### ISSUE TITLE
DINO missing

### ISSUE THREAD

**Issue by mazarinetulp** [wfanyue/DPG-T2I-Personalization#6]
Hi, Thank you so much for your work! I want to use it in my research since I am evaluating different finetuning methods for a marketing use case. I can not find the train_dreambooth_dpg_dino.py file that (as far as i interpret it) is needed to finetune based on the DINO similarity. Can you please share it, or help me to use the DINO loss function, so I can use it for my research thesis. 

Thanks so much!!
